src/components/blog/ButtonBlock/ButtonBlock.module.css

```diff
@@ -0,0 +1,14 @@
+.container {
+  align-items: center;
+  background-color: var(--color-gray-300);
+  display: flex;
+  flex-direction: column;
+  margin: 50px auto;
+  max-width: 630px;
+  padding: 50px 15px;
+}
+
+.header {
+  font: var(--font-subtitle);
+  margin: 0 0 20px;
```

Comment:
I think if we're only really trying to set one direction of margin, then the individual margin properties like `margin-bottom` would be clearer.

But for this, you could actually just set `row-gap: 20px` in the `.container` class, which would also take care of the spacing and usually do the correct thing if a row were missing.